pkg/jobs/schedule_control_test.go

```diff
@@ -216,3 +216,80 @@ func TestJobsControlForSchedules(t *testing.T) {
 		})
 	}
 }
+
+// TestFilterJobsControlForSchedules tests that a ControlJobsForSchedules query
+// does not error out even if the schedule contains a job in a state which is
+// invalid to apply the control command to. This is done by filtering out such
+// jobs prior to executing the control command.
+func TestFilterJobsControlForSchedules(t *testing.T) {
```

Comment:
I don't love this test but I haven't yet been able to think of a deterministic way to test the filtering logic without directly interacting with the `systems.Job` table to inject a job in every state.